torch/csrc/jit/graph_executor.h

```diff
@@ -47,6 +47,18 @@ TORCH_API void runRequiredPasses(const std::shared_ptr<Graph>& g);
 
 namespace detail {
 
+struct NewCoroutine : public std::exception {
```

Comment:
Both of these are unused. Let's just delete them for now. It is not clear how we will be handling the implementation of futures yet since it depends a lot on the threading library.